relational_operators/TextScanOperator.cpp

```diff
@@ -288,7 +288,7 @@ void TextScanWorkOrder::execute() {
 
     std::fclose(file);
   } else {
-    BlobReference blob = storage_manager_->getBlob(text_blob_);
+    MutableBlobReference blob = storage_manager_->getBlobMutable(text_blob_);
```

Comment:
I get the code below. Not sure about why we also need to change to MutableBlobReference. I'm sure there is a reason, and would be good to state that here.